openmls/src/credentials/codec.rs

```diff
@@ -48,6 +26,12 @@ impl tls_codec::Deserialize for Credential {
     }
 }
 
+// This is implemented manually because we call `SignaturePublicKey::new(...)`
+// and return a `DecodingError` in case when `SignaturePublicKey::new(...)` fails.
+
+// TODOs:
```

Comment:
If there's a TODO, there should be an issue number.

Reply:
Hm... I guess I wasn't sure what would be the action this TODO would ask for... Currently, this implementation could be replaced by `derive(TlsDeserialize)`. Unless we want to check the public key at this step? I could open an issue for this check and replace this implementation for now.

Reply:
Well, I don't know what you had in mind exactly when you wrote the TODO. Whatever it was, that's what should go into the issue. Otherwise, drop the TODO.

The basic credential will sort of drop out anyway, so I wouldn't worry too much.